examples/fmsynth/mmlplayer_main.c

```diff
@@ -0,0 +1,548 @@
+/****************************************************************************
+ * apps/examples/fmsynth/mmlplayer_main.c
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.  The
+ * ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ ****************************************************************************/
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <nuttx/config.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include <errno.h>
+#include <pthread.h>
+#include <sched.h>
+#include <limits.h>
+
+#include <nuttx/audio/audio.h>
+#include <audioutils/fmsynth.h>
+#include <audioutils/nxaudio.h>
+#include <audioutils/mml_parser.h>
+
+#include "operator_algorithm.h"
+#include "music_scale.h"
+#include "mmlplayer_score.h"
+
+/****************************************************************************
+ * Pre-processor
+ ****************************************************************************/
+
+#define APP_FS    (48000)
+#define APP_BPS   (16)
+#define APP_CHNUM (2)
+#define CARRIER_LEVEL (25.f / 100.f)
```

Comment:
```suggestion
#define APP_FS        (48000)
#define APP_BPS       (16)
#define APP_CHNUM     (2)
#define CARRIER_LEVEL (25.f / 100.f)
```
Also maybe `25.f / 100.f` can be replaced by `.25f`?